FreeRTOS/FreeRTOS
# Fix vTaskSuspendAll assertion unit test

Description
-----------
In this PR:
* Address the kernel PR https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/1029 to update the call order of `portGET_CORE_ID()` and `portSET_INTERRUPT_MASK()`


Test Steps
-----------
Running with the kernel PR should have no failed unit test case.

The following test cases are fixed:
>/mnt/c/msys64/home/chinglee/kernel/unit-test/FreeRTOS/FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice_mock/covg_multiple_priorities_no_timeslice_mock_utest.c:804:test_coverage_prvGetExpectedIdleTime_ready_list_eq_1:FAIL:Function ulFakePortSetInterruptMask.  Called earlier than expected.

>/mnt/c/msys64/home/chinglee/kernel/unit-test/FreeRTOS/FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice_mock/covg_multiple_priorities_no_timeslice_mock_utest.c:910:test_coverage_prvGetExpectedIdleTime_ready_list_eq_2:FAIL:Function ulFakePortSetInterruptMask.  Called earlier than expected.

>/config_assert_utest.c:587:test_prvGetExpectedIdleTime_assert_nextUnblock_lt_xTickCount:FAIL:Function ulFakePortSetInterruptMask.  Called earlier than expected.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
